h/templates/client/annotation.html

```diff
@@ -66,7 +66,7 @@
     </excerpt>
   </section>
 
-  <!-- / Excerpts -- >
+  <!-- / Excerpts -->
```

Comment:
This fixes a parse error with BeautifulSoup, which the used-class extraction tool uses. See first commit.